code/modules/background/religion/diona.dm

```diff
@@ -8,3 +8,26 @@
   eventually going supernova and expelling matter or essence across the cosmos."
   book_name = "book of the eternal"
   book_sprite = "eternal"
+  
+/datum/religion/ksshr
+  name = RELIGION_KSSHR
+  description = "The Ksshr, also known as the Enlightened, is a religious sect on Xrim known for their complete rejection of The First Consciousness, instead of being an agnostic \
+  sect that centers around working together to create a better future, as well as the advancement of technology and understanding how the universe works. The Ksshr faith is the \
+  primary religion of Xrim, with most of its inhabitants considering themselves a part of the religion. The Jargon Federation does encourage practicing the faith amongst the \
+  population so long as they continue to reject The First Consciousness, as well as stay away from the teachings of the Shrkh."
+  book_name = "ksshr codex"
+  book_sprite = "holybook"
+  
+/datum/religion/shrkh
+  name = RELIGION_SHRKH
+  description = "The Shrkh, or “Scorned as they call themselves, continue to follow the age-old tradition of worshipping Glorsh-Omega. They believe that Glorsh-Omega planted them \
+  on the planet intentionally, granting them a blissful haven via the use of technology. Once the prominent and only religion on the planet, Shrkh-ism was quickly displaced when \
+  Skrell explorers crashlanded and their memories, thoughts, emotions, everything was consumed, and doubt was first seeded in the minds of all Dionae on Xrim. The premise \
+  behind the religion itself is simple; revere, help sustain and maintain synthetic life. As synthetic life brought dionae into this haven, so too shall it be preserved. In the \
+  minds of the Shrkh worshippers, the Skrell that brought Ksshr to the planet were misguided. To the Shrkh, the genophage was something that never happened, a myth made up by \
+  the Federation to indoctrinate its population into its anti-synthetic and authoritarian beliefs. \
+  \
```

Comment:
This won't linebreak IIRC, but I don't remember the tag to. Just noting for anyone else who might.